2025-03-05 14:53:20,417 - messaging - INFO - performance_logger.py:20 - LATENCY: rabbitmq_connection took 33.17ms
2025-03-05 14:53:20,418 - messaging - INFO - subscriber.py:81 - Successfully connected to RabbitMQ
2025-03-05 14:53:20,422 - messaging - INFO - subscriber.py:121 - Started consuming messages from RabbitMQ
2025-03-05 14:53:27,708 - messaging - INFO - subscriber.py:97 - Received move: {'from_square': 'g8', 'to_square': 'h6', 'message_id': '38292e76', 'timestamp': 1741186407.7076242}
2025-03-05 14:53:27,709 - messaging - INFO - performance_logger.py:20 - LATENCY: message_delivery took 2.15ms
2025-03-05 14:53:27,711 - messaging - INFO - subscriber.py:97 - Received move: {'from_square': 'f7', 'to_square': 'f5', 'message_id': '678a7061', 'timestamp': 1741186407.7096362}
2025-03-05 14:53:27,711 - messaging - INFO - performance_logger.py:20 - LATENCY: message_delivery took 2.33ms
2025-03-05 14:53:27,843 - messaging - INFO - performance_logger.py:31 - MOVE: g8h6 FAILURE (planning: 0.00ms, execution: 82.98ms)
2025-03-05 14:53:27,843 - messaging - ERROR - performance_logger.py:35 - ERROR: movement - execution_failure: Move g8h6
2025-03-05 14:53:28,056 - messaging - INFO - performance_logger.py:31 - MOVE: f7f5 FAILURE (planning: 0.00ms, execution: 95.72ms)
2025-03-05 14:53:28,056 - messaging - ERROR - performance_logger.py:35 - ERROR: movement - execution_failure: Move f7f5
2025-03-05 14:54:31,975 - messaging - INFO - subscriber.py:97 - Received move: {'from_square': 'g8', 'to_square': 'h6', 'message_id': '7c210079', 'timestamp': 1741186471.9721472}
2025-03-05 14:54:31,976 - messaging - INFO - performance_logger.py:20 - LATENCY: message_delivery took 4.07ms
2025-03-05 14:54:31,978 - messaging - INFO - subscriber.py:97 - Received move: {'from_square': 'f7', 'to_square': 'f5', 'message_id': '1320d256', 'timestamp': 1741186471.9721472}
2025-03-05 14:54:31,978 - messaging - INFO - performance_logger.py:20 - LATENCY: message_delivery took 6.38ms
2025-03-05 14:54:32,103 - messaging - INFO - performance_logger.py:31 - MOVE: g8h6 FAILURE (planning: 0.00ms, execution: 82.36ms)
2025-03-05 14:54:32,104 - messaging - ERROR - performance_logger.py:35 - ERROR: movement - execution_failure: Move g8h6
2025-03-05 14:54:32,293 - messaging - INFO - performance_logger.py:31 - MOVE: f7f5 FAILURE (planning: 0.00ms, execution: 69.59ms)
2025-03-05 14:54:32,293 - messaging - ERROR - performance_logger.py:35 - ERROR: movement - execution_failure: Move f7f5
2025-03-05 14:56:18,132 - messaging - INFO - subscriber.py:184 - RabbitMQ connection closed
